scripts/import_srpm.py

```diff
@@ -2,12 +2,45 @@
 import argparse
 import logging
 import os
+import shutil
 import subprocess
+from glob import glob
+
 
 def call_process(args):
     logging.debug("$ %s", args)
     subprocess.check_call(args)
 
+def pipe_commands(*commands: list[str]) -> bytes:
+    if not commands:
+        raise ValueError("The 'commands' list cannot be empty.")
+    if any(not cmd for cmd in commands):
+        raise ValueError("All commands in the list must be non-empty.")
+
+    processes: list[subprocess.Popen[bytes]] = []
+    last_process_stdout = None
```

Comment:
naming it `next_process_stdin` would make it more accurate

Reply:
I suppose it depends on which side of the pipe your looking at 🙂

Reply:
in one way yes, but I'm rather seeing it as "in one case the name is always valid, while in the other there is no previous process when launching the first one"

Reply:
That's why it's a `IO[bytes] | None`.
But I'm also ok with the other name, I'll use it 👍